x-pack/plugins/cloud_security_posture/server/create_indices/create_indices.ts

```diff
@@ -140,6 +151,64 @@ const createLatestFindingsIndex = async (esClient: ElasticsearchClient, logger:
   }
 };
 
+const createLatestVulnerabilitiesIndex = async (esClient: ElasticsearchClient, logger: Logger) => {
+  try {
+    // Deletes old assets from previous versions as part of upgrade process
+    const INDEX_TEMPLATE_V830 = 'cloud_security_posture.vulnerabilities_latest';
+    await deleteIndexTemplateSafe(esClient, logger, INDEX_TEMPLATE_V830);
+
+    // We want that our latest findings index template would be identical to the findings index template
+    const vulnerabilitiesIndexTemplateResponse = await esClient.indices.getIndexTemplate({
+      name: VULNERABILITIES_INDEX_NAME,
+    });
+
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    const { template, composed_of, _meta } =
+      vulnerabilitiesIndexTemplateResponse.index_templates[0].index_template;
+
+    // We always want to keep the index template updated
+    await esClient.indices.putIndexTemplate({
+      name: LATEST_VULNERABILITIES_INDEX_TEMPLATE_NAME,
+      index_patterns: LATEST_VULNERABILITIES_INDEX_PATTERN,
+      priority: 500,
+      template: {
+        mappings: template?.mappings,
+        settings: {
+          ...template?.settings,
+          default_pipeline: latestVulnerabilitiesPipelineIngestConfig.id,
+          lifecycle: {
+            name: '',
+          },
+        },
+        aliases: template?.aliases,
+      },
+      _meta,
+      composed_of,
```

Comment:
_nit:_ lines 160-186 are repeating themselves with different parameters.
And they will probably always will be the same in both cases.
In that case, I would suggest refactoring and consolidate that